### DISCUSSION TITLE
[Bug]: Frigate+ Page Score Range selector turns to NaN

### DISCUSSION THREAD

**Discussion by binarybucks** [blakeblackshear/frigate#13025] [Report a Bug] [answered]
### Describe the problem you are having

When erasing the Range input number inside the Frigate+ Score Range selector, the number changes to NaN. 

![image](https://github.com/user-attachments/assets/d21c4972-254b-4f72-84d5-96695b8dc2c3)

The "NaN" cannot be removed by typing a number again, which is a pain on mobile because the "NaN" text has to be manually selected by the text selector again to overwrite it with a valid number instead. 

I would have expected the filed to remain empty after erasiung the old number so a new one can be typed. 

### Steps to reproduce

Open Score Range on Frigate+ Page in v0.14 GUI. Erase upper or lower bound number as one whould to quickly select another range. 

### Version

v0.14

### Frigate config file

```yaml
n/a
```

### Relevant log output

```shell
n/a
```

### Operating system

Other

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

n/a

### Any other information that may be helpful

n/a

**Comment by NickM-27** [comment, marked as answer]
fixed in https://github.com/blakeblackshear/frigate/pull/13030
